airflow/upgrade/rules/postgres_mysql_sqlite_version_upgrade_check.py

```diff
@@ -43,16 +43,23 @@ def check(self, session=None):
             min_req_sqlite_version = Version('3.15')
             installed_sqlite_version = Version(session.execute('select sqlite_version();').scalar())
             if installed_sqlite_version < min_req_sqlite_version:
-                return "From Airflow 2.0, SQLite version below 3.15 is no longer supported. \n" + more_info
+                return "From Airflow 2.0, SQLite version below {} is no longer supported. \n{}".format(
+                    min_req_sqlite_version, more_info
+                )
 
         elif "postgres" in conn_str:
             min_req_postgres_version = Version('9.6')
             installed_postgres_version = Version(session.execute('SHOW server_version;').scalar())
             if installed_postgres_version < min_req_postgres_version:
-                return "From Airflow 2.0, PostgreSQL version below 9.6 is no longer supported. \n" + more_info
+                return "From Airflow 2.0, PostgreSQL version below {} is no longer supported. \n{}".format(
+                    min_req_postgres_version, more_info
+                )
 
         elif "mysql" in conn_str:
             min_req_mysql_version = Version('5.7')
-            installed_mysql_version = Version(session.execute('SELECT VERSION();').scalar())
+            # special treatment is needed here, because MySQL version may include a suffix like '-log'
+            installed_mysql_version = Version(session.execute('SELECT VERSION();').scalar().split('-')[0])
             if installed_mysql_version < min_req_mysql_version:
-                return "From Airflow 2.0, MySQL version below 5.7 is no longer supported. \n" + more_info
+                return "From Airflow 2.0, MySQL version below {} is no longer supported. \n{}".format(
```

Comment:
Upgrade check needs to be fixed in v1-10-stable too

https://github.com/apache/airflow/blob/v1-10-stable/airflow/upgrade/rules/postgres_mysql_sqlite_version_upgrade_check.py#L54-L58

Reply:
Hi Kaxil, I'm not sure if I get you correctly, but this PR is raised against v1-10-stable, so it should be doing what you are suggesting.
This `airflow/upgrade` directory does not exist in `master` branch anyway :)

Reply:
Whoops my bad!